views/list.js

```diff
@@ -73,11 +73,17 @@ ListView.prototype.createElement = function (props) {
       <div>
         ${props.features.sort(cmpDate).map(function (f) {
           const fprops = f.properties
+          const match = props.features.filter(feat => f.properties._id === feat.properties._id)
+          //If there are no-coordinates to zoom to, does nothing
+          const hasCoords =  (Array.isArray(match[0].geometry.coordinates)
+            && match[0].geometry.coordinates[0] != undefined 
+            && match[0].geometry.coordinates[1] != undefined)
+         
           return html`
-          <button class='item' onclick=${props.onClick.bind(
+          <button class='item' onclick=${hasCoords ? props.onClick.bind(
             null,
             f.properties._id
-          )}>
+          ) :"return"}>
```

Comment:
does `null` or `undefined` work here? can't remember the internals of nanohtml too well 😅

Reply:
Actually I suppose it would make more sense to use `''` here, since its a templating language

Reply:
wait, im confusing the html with the button binding property

Reply:
No, I think it does make sense to use `return` because it is templating into `button onClick=return'. I can't find anything in the nanohtml documentation that says what to do in this case. But it seems to b working with return.

Reply:
adding "return" as a string will leave it in the html which we don't want. if you provide a falsy value, nanohtml should omit that handler as a whole (similar to jsx)

Reply:
Ok! sounds good